.gitignore

```diff
@@ -1,2 +1,6 @@
+<<<<<<< HEAD
+node_modules
+=======
 /node_modules/
 .env
+>>>>>>> dfc20d92b8f8a94d25d8d3067e24e1c4d470505f
```

Comment:
seems like the old problem accurred, please fix

Reply:
Ok, I fix it now.